scripts/python/test_add_users_everyone_github_team.py

```diff
@@ -0,0 +1,26 @@
+import sys
+import unittest
+from unittest.mock import patch
+
+from github import Github
+from gql import Client
+from gql.transport.aiohttp import AIOHTTPTransport
+
+import add_users_everyone_github_team
+
+
+@patch.object(Github, "__new__")
+@patch.object(AIOHTTPTransport, "__new__")
+@patch.object(Client, "__new__")
+class TestAddUsersEveryoneGithubTeam(unittest.TestCase):
+
+    def test_main_smoke_test(self, mock1, mock2, mock3):
```

Comment:
What does this test achieve? As no assert.
Does mock1,2,3 relate to the three patch object above? And if so is it based on order of the patch objects?

Reply:
The idea of it is to provide a minimal testing bed for the script. The [smoke test](https://www.techtarget.com/searchsoftwarequality/definition/smoke-testing#:~:text=Smoke%20testing%2C%20also%20called%20build,build%20and%20before%20a%20release.) usually have no assertions and are mainly there to ensure the application/script can be loaded appropriately.

This will help prevent the issue you spotted yesterday with my dodgy importing 😂 

As we increase our unit/integration tests, the smoke test will be of minimal value and can be removed 🔥 

And yes the `mock1,mock2,mock3` relate to the patched objects (which I believe are passed through as `MagicMock` objects. The ordering is a bit odd (backwards to what I'm used to) - so thought I'd leave them with an ambiguous name until we add unit/integration tests to use them and I'm more certain as to which object they line up with 😂

Reply:
Cool ok sounds good